PWGHF/TableProducer/treeCreatorLcToPKPi.cxx

```diff
@@ -203,8 +205,12 @@ struct HfTreeCreatorLcToPKPi {
     // Filling event properties
     rowCandidateFullEvents.reserve(collisions.size());
     for (const auto& collision : collisions) {
+      if (!collision.has_mcCollision()) {
+        continue;
```

Comment:
@DelloStritto Do we really want to silently drop these? Wouldn't it make sense to keep them and store -1 for the MC collision ID?

Reply:
@qgp Where would we use these collisions without a matching reconstruction level?